### DISCUSSION TITLE
Cannot use jose on Angular application

### DISCUSSION THREAD

**Discussion by YounesJonas** [panva/jose#482] [Q&A]
Hi All,

I've added jose (4.11.1) as a dependency to my angular project, (Angular CLI 1.4.7, Node 14.17.6, typescript 2.5.3)

getting the following error when compiling : (ng serve)

`
// import
const jose = require('jose');
export class AuthService {

public decryptToken(){
const decryptedJWE = jose.jwtDecrypt(jwe,secret);
}
}
`
I got this error:

```
Error in ./node_modules/jose/lib/jwk/key/base.js
Module parse failed: Unexpected token(101:8)
You may need an appropriate loader to handle this file type.
```
Has anyone encountered it before?

Best regards,

Thanks

**Comment by panva** [comment]
Would you mind creating a minimal reproduction repository and steps to reproduce your issue? For anyone without prior knowledge of angular, a repo that can be simply cloned, npm installed and ran is needed to help you.
